### PR TITLE
Azure CI: Enable ARM plugin build

### PR DESCRIPTION
**Pull request by asuhov**

### CODE REVIEW

**Review by azhogov — APPROVED** [review]

File: .ci/azure/linux_arm64.yml [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+  - task: CMake@1
+    inputs:
+      cmakeArgs: >
+        -GNinja
+        -DVERBOSE_BUILD=ON
+        -DCMAKE_BUILD_TYPE=$(BUILD_TYPE)
+        -DCMAKE_TOOLCHAIN_FILE=$(OPENVINO_REPO_DIR)/cmake/arm64.toolchain.cmake
+        -DOpenCV_DIR=$(BUILD_DIR_OPENCV)/install/lib/cmake/opencv4
+        -DENABLE_OPENCV=OFF
```
Comment (ilya-lavrenov) [inline .ci/azure/linux_arm64.yml:103]: why OPENCV is OFF while provide path to OpenCV?

File: .ci/azure/linux_arm64.yml [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+        -DENABLE_SSE42=OFF
+        -DENABLE_MYRIAD=OFF
+        -DENABLE_GNA=OFF
```
Comment (ilya-lavrenov) [inline .ci/azure/linux_arm64.yml:106]: `ENABLE_GNA` and `ENABLE_SSE42` are not necessary. They are off by default

File: .ci/azure/linux_arm64.yml (hunk 1 above)
Comment (asuhov) [reply .ci/azure/linux_arm64.yml:103]: It's a workaround to disable automatic OpenCV package downloading. Copied from https://github.com/openvinotoolkit/openvino_contrib/blob/master/modules/arm_plugin/Dockerfile.RPi64#L47

File: .ci/azure/linux_arm64.yml (hunk 2 above)
Comment (asuhov) [reply .ci/azure/linux_arm64.yml:106]: Thanks, will remove. Need to remove here as well: https://github.com/openvinotoolkit/openvino_contrib/blob/master/modules/arm_plugin/Dockerfile.RPi64#L49

**Pull request merged**